tests/integration/components/actions-test.js

```diff
@@ -12,11 +12,12 @@ test('It supports deprecated lifecycle actions', function(assert) {
   // onTooltip____ actions are deprecated in favor of on on_____ actions
   // these actions will be supported until v3.0.0
 
+  // actions can have unique names
```

Comment:
I added unique action names because when I initially made my changes compliant with this test I wrote a solution dependent on the action names being `onShow`, `onHide`, etc.